heartbeat/docs/getting-started.asciidoc

```diff
@@ -96,9 +96,43 @@ was started. Heartbeat adds the `@every` keyword to the syntax provided by the
 
 include::{libbeat-dir}/shared/config-check.asciidoc[]
 
+[float]
+[[configurelocation]]
+=== Step 4: Configure the Heartbeat location
+
+Heartbeat can be deployed in multiple locations so that you can detect 
+differences in availability and response times across those locations.
+By configuring Heartbeat with the location of the server you will allow the
+Uptime application to display location specific information on a map, and 
+allow the Uptime anomaly detection to detect anomalies based on location.
```

Comment:
```suggestion
Configure the Heartbeat location to allow {kib} to display location-specific
information on Uptime maps and perform Uptime anomaly detection based
on location.
```

Reply:
I'm trying to shorten this a bit and use language that's parallel with the heading, but my editing brain's a bit out to lunch. Is this better?

Reply:
much better! Thanks.